src/m365/planner/commands/task/task-list.ts

```diff
@@ -161,6 +180,17 @@ class PlannerTaskListCommand extends GraphItemsListCommand<Task> {
       });
   }
 
+  private mergeTaskPriority(taskItems: any[], betaTaskItems: any[]): any[] {
+    const findBetaTask = (id: string) => betaTaskItems.find(task => task.id === id);
+
+    taskItems.forEach(task => {
+      const { priority } = findBetaTask(task.id);
```

Comment:
Theoretically it's possible, that we won't find a matching task. In such case, this will lead to an unhandled exception on runtime. Let's be a bit more defensive and take it into account.